### PR TITLE
Ruby Capstone : Catalog of things

### PR DESCRIPTION
**Pull request by ohyajk**
## In this pr we did the following :

- Insert and query data in SQL.
- Use primary key & foreign key mechanisms for joining tables.
- Understand the different types of relationships between tables.
- Query multiple tables.
- Prepare complex queries that answer analytical questions.
- Run a program using the command line.
- Use Ruby syntax for basic programming operations.
- Apply Ruby best practices and language style guides in code.
- Describe the SOLID principles of OOP.
- Implement classes and objects in Ruby.
- Understand the four main principles of OOP.
- Implement encapsulation and inheritance with Ruby.
- Create a UML class diagram.
- Explain the difference between associations, aggregations, and composition in OOP.
- Set up associations between classes and objects.
- Store data in files.
- Build interactive console apps.
- Recognize the value of making equal contributions to group projects to produce the best outcome.

### CODE REVIEW

File: create_items.rb [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+  def item_options
+    puts 'which item would you like to create?'
+    puts '1. Add a Book'
+    puts '2. Add a Music Album'
+    puts '3. Add a Game'
+  end
+
+  def add_selected_item
+    # @item_options = gets.chomp
```
Comment (Rustamxon7) [inline create_items.rb:19]: **You all did great job. Well implemented methods and tests.**

- [ ] However, I think this one was just out of your attention due to some reasont. Do not worry, it's not the end of the world. You are free to fix this up. If this commented code is not necessary action in this project. Please, remove it. Thank you!

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello Team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Great implemented tests_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: create_items.rb (hunk 1 above)
Comment (Tekhlay) [reply create_items.rb]: ### Thanks We have resolve it!

**Review by Ismailco — APPROVED** [review]
### Hi @ohyajk, @Tekhlay, @houdamzari  

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

cr-approve

## Highlights

✔️ Clean UI
✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

Good luck! 🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**